akka-http-core/src/main/scala/akka/http/javadsl/model/RequestResponseAssociation.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.http.javadsl.model
+
+import java.util.concurrent.CompletableFuture
+
+import akka.annotation.ApiMayChange
+
+import scala.concurrent.Promise
+
+/**
+ * A marker trait for attribute values that should be (automatically) carried over from request to response.
+ */
+@ApiMayChange
+trait RequestResponseAssociation extends akka.http.scaladsl.model.RequestResponseAssociation
```

Comment:
Usually we have the dependency hierarchy the other way round. Would that work as well?

Reply:
Sure, I think I just misremembered and didn't double check. I'll fix!